jsonld/credit-classes/C01-verified-carbon-standard-class.json

```
@@ -0,0 +1,56 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "dash": "http://datashapes.org/dash#",
+    "schema:itemListElement": { "@container": "@list" }
+  },
+  "@type": "regen:CreditClass",
+  "schema:description": "",
+  "regen:sourceRegistry": {
+    "schema:name": "",
+    "schema:url": {
+      "@type": "schema:URL",
+      "@value": ""
+    }
+  },
+  "regen:sectoralScope": [
+    {
+      "@type": "xsd:string",
+      "@value": ""
+    }
```

no need to specify the type here, just a string is enough and should work fine with the validation
```suggestion
""
```

---

got it, thanks for pointing this out..